sdk/highlight-ex/mix.exs

```diff
@@ -0,0 +1,33 @@
+defmodule Highlight.MixProject do
+  use Mix.Project
+
+  def project do
+    [
+      app: :highlight,
+      version: "0.1.0",
```

Comment:
how do we distribute this package? anything we need to add to CI to publish this SDK automatically?